### PR TITLE
Some bugs about multiple implemented interfaces

### PR DESCRIPTION
**Pull request by pepakriz**

### CODE REVIEW

File: SlevomatCodingStandard/Helpers/ReferencedNameHelper.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ private static function createAllReferencedNames(PHP_CodeSniffer_File $phpcsFile
 								array_merge(TokenHelper::$nameTokenCodes, TokenHelper::$ineffectiveTokenCodes),
 								$previousTokenBeforeStartPointer - 1
 							);
-							if ($tokens[$implementsPointer]['code'] !== T_IMPLEMENTS) {
+							if ($tokens[$implementsPointer]['code'] !== T_IMPLEMENTS && $tokens[$implementsPointer]['code'] !== T_EXTENDS) {
```
Comment (ondrejmirtes) [inline SlevomatCodingStandard/Helpers/ReferencedNameHelper.php]: Přejmenovat na `$implementsOrExtendsPointer`

File: SlevomatCodingStandard/Helpers/ReferencedNameHelper.php [hunk 2]
```diff
@@ -92,11 +92,14 @@ private static function createAllReferencedNames(PHP_CodeSniffer_File $phpcsFile
...
-							$implementsPointer = TokenHelper::findPreviousExcluding(
-								$phpcsFile,
-								array_merge(TokenHelper::$nameTokenCodes, TokenHelper::$ineffectiveTokenCodes),
-								$previousTokenBeforeStartPointer - 1
-							);
+							$implementsPointer = $previousTokenBeforeStartPointer;
+							do {
+								$implementsPointer = TokenHelper::findPreviousExcluding(
+									$phpcsFile,
+									array_merge(TokenHelper::$nameTokenCodes, TokenHelper::$ineffectiveTokenCodes),
```
Comment (ondrejmirtes) [inline SlevomatCodingStandard/Helpers/ReferencedNameHelper.php]: Nešlo by to jen přes `array_merge([T_COMMA], TokenHelper::$nameTokenCodes, TokenHelper::$ineffectiveTokenCodes),` ?

File: SlevomatCodingStandard/Helpers/ReferencedNameHelper.php (hunk 1 above)
Comment (pepakriz) [inline SlevomatCodingStandard/Helpers/ReferencedNameHelper.php]: done

**Comment by ondrejmirtes** [comment]
Thanks! 👍

**Pull request merged**